soco/data_structures.py

```diff
@@ -598,7 +650,7 @@ class DidlAudioItem(DidlItem):
 
     """An audio item."""
 
-    item_class = 'object.item.audioItem'
+    item_class = 'object.item.audioitem'
```

Comment:
I think something went wrong with some rebasing as this reverts you misspelling bug fix